apps/studio/src/features/dashboard/components/CollectionTable/CollectionTableMenu.tsx

```diff
@@ -33,19 +33,20 @@ export const CollectionTableMenu = ({
       />
       <Portal>
         <MenuList>
-          {resourceType === ResourceType.CollectionPage && (
-            <MenuItem
-              icon={<BiCog fontSize="1rem" />}
-              onClick={() =>
-                setPageSettingsModalState({
-                  pageId: resourceId,
-                  type: resourceType,
-                })
-              }
-            >
-              Edit page settings
-            </MenuItem>
-          )}
+          {resourceType === ResourceType.CollectionPage ||
+            (resourceType == ResourceType.CollectionLink && (
+              <MenuItem
+                icon={<BiCog fontSize="1rem" />}
+                onClick={() =>
+                  setPageSettingsModalState({
+                    pageId: resourceId,
+                    type: resourceType,
+                  })
+                }
+              >
+                Edit page settings
+              </MenuItem>
+            ))}
```

Comment:
instead of
```ts
{resourceType === ResourceType.CollectionPage ||
  (resourceType == ResourceType.CollectionLink && (...)
 )}
```

should the bracket be around the entire condition instead?
```ts
{(resourceType === ResourceType.CollectionPage || resourceType === ResourceType.CollectionLink) && (...)}
```

Reply:
ah yes, ps, thanks for the catch!